### ISSUE TITLE
Select index page size (number of records) with friendly UI control

### ISSUE THREAD

**Issue by chrisinmtown** [MISP/MISP#7151]
Let me choose the number of events shown on the event index page (and all index pages).  The events index page,  attributes index page, logs index page and others set a fixed limit of 60.  Just to be different, the tags index page sets a fixed limit of 50 and the proposals index page sets a fixed limit of 20.

**Describe the solution you'd like**

Add a drop-down selection box at the bottom of *every* index screen with page-size (record count) choices like 25, 50, 100, 250, 500

**Describe alternatives you've considered**

Probably there's a secret parameter you can add to the URL but I didn't figure it out.

**Comment by iglocska** [comment]
page:1/limit:100 should work for example

**Comment by chrisinmtown** [comment]
Thanks @iglocska that's good news, the server side already supports variable parameters, the UI just needs a new control so the user can pick a value easily.

**Comment by chrisinmtown** [comment]
Closely related, I need to choose a *small* page size to view an event that contains many MISPObjectAttribute collections.  Trying to view an event with 60 of those, where each obj-attr has 1..100 attributes, floods the browser with too much data and it becomes very slow. You want a new issue for that?

**Comment by iglocska** [comment]
Indeed, I think we have two action items here that we should take care of, though one is not going to help directly right away:

- [ ] Add a page limit dropdown to the generic index UI
- [ ] Finally get my lazy self to finish the alternate event view for large events (split attribute + object lists)

We are in the process of slowly migrating all of our views to the new framework we have for view generation, but we're at about 20% done with moving the views, so the first option there won't help in the case of a lot of index views just yet.
